config/mcu_definitions/toolhead/BTT_SB2209_v1.0.cfg

```diff
@@ -0,0 +1,22 @@
+[board_pins toolhead_manufacturer]
+mcu: toolhead
+aliases:
+   MCU_TMCDRIVER_STEP=PD0 , MCU_TMCDRIVER_DIR=PD1 , MCU_TMCDRIVER_ENABLE=PD2 , MCU_TMCDRIVER_UART=PA15 ,
+   MCU_TMCDRIVER_DIAG=PB3 ,
+
+   MCU_ENDSTOP1=PB6 , MCU_ENDSTOP2=PB5 , MCU_ENDSTOP3=PB7 ,
+   MCU_PROBE=PB8 ,
+   MCU_SERVOS=PB9 ,
+
+   MCU_HOTEND0=PB13 ,
+   MCU_TH0=PA3 ,
+   
+   MCU_FAN1=PA0 , MCU_FAN2=PA1 ,
```

Comment:
This is apparently `MCU_FAN1_PWM` and `MCU_FAN2_PWM` (I don't really like it but it's what is in the doc)